docs/scripting-ref.rst

```diff
@@ -928,29 +932,71 @@ Example:
          return {png=splash:png()}
      end
 
+.. _splash-get-viewport-size:
 
-.. _splash-set-viewport:
+splash:get_viewport_size
+------------------------
 
-splash:set_viewport
--------------------
+Get the browser viewport size.
+
+**Signature:** ``width, height = splash:get_viewport_size()``
+
+**Returns:** two numbers: width and height of the viewport in pixels.
+
+
+.. _splash-set-viewport-size:
+
+splash:set_viewport_size
+------------------------
 
-Set the browser viewport.
+Set the browser viewport size.
 
-**Signature:** ``width, height = splash:set_viewport(size)``
+**Signature:** ``splash:set_viewport_size(width, height)``
 
 **Parameters:**
 
-* size - string, width and height of the viewport.
-  Format is ``"<width>x<heigth>"``, e.g. ``"800x600"``.
-  It also accepts ``"full"`` as a value; ``"full"`` means that the viewport size
-  will be auto-detected to fit the whole page (possibly very tall).
+* width - integer, requested viewport width in pixels;
+* height - integer, requested viewport height in pixels.
+
+This will change the size of the visible area and subsequent rendering
+commands, e.g. :ref:`splash-png`, will produce an image with the specified
```

Comment:
Hmm, is it not a parenthetical expression? I did miss a comma though, according to s&w e.g. must be surrounded with commas from both sides.